libr/core/cmd_debug.c

```diff
@@ -3172,19 +3236,21 @@ static int cmd_debug(void *data, const char *input) {
 			} else eprintf ("Injection opcodes so long\n");
 			}
 			break;
-		default:{
-			const char* help_msg[] = {
-			"Usage: dx", "", " # Code injection commands",
-			"dx", " <opcode>...", "Inject opcodes",
-			"dxa", " nop", "Assemble code and inject",
-			"dxe", " egg-expr", "compile egg expression and inject it",
-			"dxr", " <opcode>...", "Inject opcodes and restore state",
-			"dxs", " write 1, 0x8048, 12", "Syscall injection (see gs)",
-			"\nExamples:", "", "",
-			"dx", " 9090", "Inject two x86 nop",
-			"\"dxa mov eax,6;mov ebx,0;int 0x80\"", "", "Inject and restore state",
-			NULL};
-			r_core_cmd_help (core, help_msg);
+		case '?':
+		default:
+			{
+				const char* help_msg[] = {
```

Comment:
If we do so we should do it in several more places. So dont take this suggestion much seriously unless the change makes sense because the help must be shown from more than one place. Which is not the case